gtk/gtk-play.c

```diff
@@ -55,6 +59,9 @@ typedef struct
   GtkWidget *media_info;
   gulong seekbar_value_changed_signal_id;
   gboolean playing;
+  GdkPixbuf *cover_pixbuf;
+  GstSample *cover_sample;
+  GMutex lock;
```

Comment:
ah got it.. should have reminded myself that  its gtk app and not gstplayer lib :)